index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Ivan Martinez - Portfolio</title>
+   <link rel="stylesheet" href="css/styles.css">
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> 
+   <link rel="preconnect" href="https://fonts.googleapis.com">
+   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700;900&display=swap" rel="stylesheet">
+</head>
+<body>
+   <section class="sec-portada">
+      <header> 
+        <a href="./index.html" class="logo">My Logo</a>
+         <nav>
+            <i id="id-menu-hamburger" class="menu-hamburg fa fa-bars"></i>
+            <ul class="menu">
+               <li class="menu-option">Hello</li>
+               <li class="menu-option">Portfolio</li>
+               <li class="menu-option">About</li>
+               <li class="menu-option">Contact</li>
+            </ul>
+         </nav>
+      </header>
+      
+      <section id="port-without-header">
```

Comment:
Greate job! you're almost done with this project, but just make sure to apply the bellow changes on it so that you can begin the next task.

- [ ] As the previous reviewer mentioned, it's very important to follow the exact Figma design principles, so please try to make sure you're using the same font type, and text size.

- [ ] You can also update the background color which is white in the Figma design.


![image](https://user-images.githubusercontent.com/34779928/192012490-4377c764-3641-4a56-baa7-8191adcc44ea.png)